src/page.ts

```diff
@@ -35,7 +35,17 @@ export interface DappeteerPage<P = unknown> {
   ): Promise<Response>;
   waitForSelector(
     selector: string,
-    opts?: Partial<{ visible: boolean; timeout: number }>
+    opts?: Partial<{
+      visible: boolean;
+      detached: boolean;
+      hidden: boolean;
+      timeout: number;
+    }>
+  ): Promise<DappeteerElementHandle>;
+
+  waitForSelectorIsGone(
+    selector: string,
+    opts?: Partial<{ timeout: number }>
   ): Promise<DappeteerElementHandle>;
```

Comment:
Why do we return an element if waiting for the element is gone?
Expected to get a `boolean`, for like if is gone or still available

Reply:
yup, should be removed